docs/asyncio_categorizing_functions.md

```diff
@@ -12,7 +12,7 @@ Coroutines are special functions based on Python’s generators syntax which all
 
 Invoking a coroutine function will return a Generator object back, but will not actually begin execution. This object will execute the task when it is either yielded from (from within another coroutine) or it is scheduled on the event loop.
```

Comment:
Should we change the language here to using `awaited` instead of `yielded from`?

Reply:
I agree :) done.

Reply:
That was a quick review!